docs/ops/movement/ShuffleChannels_1.md

```diff
@@ -8,43 +8,57 @@
 
 **Short description**: *ShuffleChannels* permutes data in the channel dimension of the input tensor.
 
+**Detailed description**:
+
+Input ND tensor of `data_shape` is always interpreted as 4D tensor with the following shape:
+
+    dim 0: data_shape[0] * data_shape[1] * ... * data_shape[axis-1] (or 1 if axis == 0)
+    dim 1: group
+    dim 2: data_shape[axis] / group
+    dim 3: data_shape[axis+1] * data_shape[axis+2] * ... * data_shape[data_shape.size()-1] (or 1 if axis points to last dimension)
+
+So the dimensions before and after the dimension pointed by `axis` are flattened and reshaped back to the original shape after channnels shuffling.
+
+
+The operation is equivalent with the following transformation of the input tensor *x* of shape *[N, C, H, W]*:
+
+\f[
+x' = reshape(x, [N, group, C / group, H * W])\\
+x'' = transpose(x', [0, 2, 1, 3])\\
+y = reshape(x'', [N, C, H, W])\\
+\f]
+
+where `group` is the layer attribute described below and the `axis = 1`.
+
 **Attributes**:
 
 * *axis*
 
   * **Description**: *axis* specifies the index of a channel dimension.
-  * **Range of values**: an integer number in the range [-4, 3]
+  * **Range of values**: an integer number in the range `[-rank(data_shape), rank(data_shape) - 1]`
   * **Type**: `int`
   * **Default value**: 1
   * **Required**: *No*
 
 * *group*
 
   * **Description**: *group* specifies the number of groups to split the channel dimension into. This number must evenly divide the channel dimension size.
-  * **Range of values**: a positive integer
+  * **Range of values**: a positive integer in the range `[1, data_shape[axis]]`
   * **Type**: `int`
   * **Default value**: 1
   * **Required**: *No*
 
 **Inputs**:
 
-*   **1**: 4D input tensor of any supported data type. Required.
+*   **1**: `data` ND (1D or bigger) input tensor of type *T*. **Required.**
```

Comment:
do not use word `ND`. Rewrite like this:
```
`data` input tensor  of type *T* and rank greater or equal to 1.
```

Reply:
Done